sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -172,6 +172,88 @@
       "addons": [],
       "id": "elk7map"
     },
+    {
+      "label": "HTML",
+      "labelWidth": "",
+      "labelMargin": "",
+      "tag": "p",
+      "className": "",
+      "attrs": [
+        {
+          "attr": "",
+          "value": ""
+        }
+      ],
+      "content": "Please ensure your information below matches your identity verification form.",
+      "refreshOnChange": true,
+      "customClass": "",
+      "hidden": false,
+      "modalEdit": false,
+      "key": "html9",
+      "tags": [],
+      "properties": {},
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": "",
+        "json": ""
+      },
+      "customConditional": "show = data.identityProvider === \"bcsc\" && data.mode !== \"student-edit\";",
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
```

Comment:
The content here `Please ensure your information below matches your identity verification form.` is specific to BCeid according to the existing code. Hence it is not required to be created for BCSC as bcsc never had this before. 
![image](https://github.com/user-attachments/assets/ddebf2f5-f2c5-4650-84a1-d1dbccdee2c4)

Hence please remove this.